src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -9191,6 +9627,183 @@ .maxstack  2
 ");
         }
 
+        [Theory]
+        [CombinatorialData]
+        public void ConsumeAbstractBinaryOperatorForTupleEquality_03([CombinatorialValues("==", "!=")] string op)
+        {
+            var source1 =
+@"
+public interface I1<T> where T : I1<T>
+{
+    abstract static bool operator == (T x, T y);
+    abstract static bool operator != (T x, T y);
+}
+
+class Test
+{
+    static void M02<T, U>((int, T) x) where T : U where U : I1<T>
```

Comment:
I was surprised that this worked. I thought `T` needed to be directly constrained to `I1<T>` to access abstract static members on it for the time being.

Reply:
> I was surprised that this worked. I thought T needed to be directly constrained to I1<T> to access abstract static members on it for the time being.

This isn't an operator, whih have signature constraints, and   it is not declared in an interface.